src/test/regress/sql/columnar_drop.sql

```diff
@@ -15,23 +15,27 @@
 -- store postgres database oid
 SELECT oid postgres_oid FROM pg_database WHERE datname = 'postgres' \gset
 
-SELECT count(distinct storage_id) AS columnar_stripes_before_drop FROM columnar.stripe \gset
-
 -- DROP columnar tables
-DROP TABLE contestant;
-DROP TABLE contestant_compressed;
+SET client_min_messages TO WARNING;
+DROP TABLE IF EXISTS contestant;
+DROP TABLE IF EXISTS contestant_compressed;
+RESET client_min_messages;
 
 -- make sure DROP deletes metadata
-SELECT :columnar_stripes_before_drop - count(distinct storage_id) FROM columnar.stripe;
+WITH cte AS (SELECT distinct relation as distinct_rel FROM columnar.stripe)
+SELECT
+  (SELECT COUNT(*) > 0 FROM cte WHERE distinct_rel::regclass::text like '%contestant%') AS contestant_rel_exists,
+  (SELECT COUNT(*) > 0 FROM cte WHERE distinct_rel::regclass::text like '%contestant_compressed%') AS contestant_compressed_rel_exists;
```

Comment:
This PR needs to update output of this test hence this brings the necessity of making this test idempotent. Those changes are done to make this test idempotent.

Reply:
That's also why this PR updates run_test.py.